Cargo.toml

```diff
@@ -3,6 +3,11 @@ name = "openmls"
 version = "0.2.0"
 authors = ["Raphael Robert"]
```

Comment:
Authors should change to something like "The openmls developers <some@email.com>"

Reply:
Agreed, as I'll change it to that as soon as we have a DL set up.

Reply:
I also added that to #196.